doc/source/science_guide/sg_boundary_forcing.rst

```diff
@@ -143,6 +144,10 @@ respectively, given the ice velocity :math:`\vec{U}_i`:
    \end{aligned}
    :label: stars
 
+
+Note that *atmo_boundary_layer* also accepts an optional argument, ``zlvs``, to support receiving scalar quantities from the atmospheric model (humidity and temperature)
+at a different vertical level than the winds. In that case a separate stability :math:`\Upsilon_s` is computed using the same formula as above but substituting :math:`z_o` by :math:`z_{o,s}`.
```

Comment:
The "staggered" terminology makes sense here in the way that you use it, but it would be helpful to just add `(staggered)` after `at a different vertical level than the winds`, to more directly connect the comment below with this one.  Optional.

Reply:
yes, good idea.

Reply:
I've just pushed an updated commit that now reads 

> Note that *atmo_boundary_layer* also accepts an optional argument, ``zlvs``, to support staggered atmospheric levels, i.e. receiving scalar quantities from the atmospheric model (humidity and temperature)at a different vertical level than the winds.